spec/component/models/user_spec.rb

```diff
@@ -421,7 +421,10 @@
 
   describe '.needs_open_access_notification' do
     # Users who meet all of the criteria to receive an email
-    let!(:email_user_1) { create :user, open_access_notification_sent_at: 1.year.ago, first_name: 'email_user_1' }
+    let!(:email_user_1) { create :user,
+                                 open_access_notification_sent_at: 1.year.ago,
+                                 first_name: 'email_user_1',
+                                 psu_identity: { 'data' => { 'link' => '', 'cprid' => '', 'active' => true, 'userid' => '', 'confHold' => false, 'givenName' => '', 'familyName' => '', 'affiliation' => ['FACULTY', 'MEMBER'], 'displayName' => '', 'serviceAccount' => false, 'universityEmail' => '' } } }
```

Comment:
```suggestion
 psu_identity: { 'data' => { 'link' => '', 
                              'cprid' => '', 
                              'active' => true, 
                              'userid' => '', 
                              'confHold' => false, 
                              'givenName' => '', 
                              'familyName' => '', 
                              'affiliation' => ['FACULTY', 'MEMBER'], 
                              'displayName' => '', 
                              'serviceAccount' => false, 
                              'universityEmail' => '' }
                     } 
               }
```
For hashes, it's sometimes helpful to add a new line for each key/value and align them so they don't run off the screen.